lib/mirage.ml

```diff
@@ -2380,7 +2380,11 @@ let configure_opam t =
         | major::minor::_ ->
           let major = try int_of_string major with Failure _ -> 0 in
           let minor = try int_of_string minor with Failure _ -> 0 in
-          if (major, minor) >= (1, 2) then opam "install" ps else version_error ()
+          if (major, minor) >= (1, 2) then (
+            opam "install" ["depext"];
```

Comment:
I like this feature, but I don't like that it attempts to install depext on every `mirage` invocation.  Isn't there a more lightweight way to check if `depext` is installed than calling the solver every time? (`opam install` is slow on ARM).  I'd also like a flag or Makefile variable that skips DEPEXT installation for systems where it's not relevant.